src/features/import-maps.js

```diff
@@ -39,6 +39,9 @@ function processScripts () {
       script.sp = true;
       var fetchPromise = script.src ? fetch(script.src, { integrity: script.integrity }).then(function (res) {
         return res.text();
+      }).catch((err) => {
+        console.warn(Error(errMsg('W4', process.env.SYSTEM_PRODUCTION ? script.src : 'Failed to fetch ' + script.src)));
```

Comment:
I'm still just a little bit concerned about the error swallowing I must admit, when those errors might still contain useful information.

I'm wondering perhaps we should just decorate the existing error:

Eg:

```suggestion
        err.message = errMsg('W4', process.env.SYSTEM_PRODUCTION ? script.src : 'Failed to fetch ' + script.src) + '\n' + err.message);
        console.warn(err);
```

Do you think the output might look ok with that? Or do these errors still show up in the console anyway for inspection?

For the JSON parsing error I think the current handling is fine, since we include the reinterpretation of the context and will likely improve this in due course anyway.

Reply:
I've pushed a change to decorate the error including the status code as well.